### DISCUSSION TITLE
Secondary labels for each fields on edit view

### DISCUSSION THREAD

**Discussion by krystof-k** [avo-hq/avo#2578] [Help] [answered]
I'm getting labels (`<label for…`) above every input for each field on edit view.

![Snímek obrazovky 2024-03-09 v 17 38 57](https://github.com/avo-hq/avo/assets/11633481/7311819d-e844-4ee4-96fd-2b40d2fa21e4)

Is this intentional?

**Comment by adrianthedev** [comment]
Nope. Can you please open an issue and fill in the issue template fields?
Thanks!

**Reply by krystof-k** [reply]
Will do, hope this is not something environment-related.

**Reply by adrianthedev** [reply]
We'll see. I never saw that before, and it definitely should not be like that.
Please add your ViewComponent version as well.

**Comment by krystof-k** [comment, marked as answer]
Just linking the issue: #2604.

**Reply by adrianthedev** [reply]
Thanks for posting the fix!
